### PR TITLE
Issue 32218, added more details on locating login credentials

### PR DESCRIPTION
**Pull request by ahardin-rh**
Addresses https://github.com/openshift/openshift-docs/issues/32218

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@spadgett PTAL. Thanks!

File: modules/web-console-overview.adoc [hunk 1]
```diff
@@ -20,3 +17,5 @@ INFO Login to the console with user: kubeadmin, password: <provided>
 ----
 
 Use those details to log in and access the web console.
+
+For existing clusters that you did not install, you can use `oc whoami --show-console` to locate the login credentials.
```
Comment (spadgett) [inline modules/web-console-overview.adoc:21]: This doesn't actually locate the credentials. It only prints the URL of the console.

**Review by spadgett — APPROVED** [review]
lgtm

**Review by jeana-redhat — APPROVED** [review]

**Pull request merged**